src/orange/orangeinp/UnitProto.hh

```diff
@@ -134,7 +135,8 @@ class UnitProto : public ProtoInterface
         //! For non-global units, assume inside the boundary
         bool remove_interior{true};
         //! Use DeMorgan's law to remove negated joins
-        bool remove_negated_join{false};
+        bool remove_negated_join{orange_tracking_logic()
+                                 == LogicNotation::infix};
```

Comment:
Wondering if we should keep this option as it must be enabled with infix tracking. Or just add a runtime check